cypress/e2e/course_board/shareBoardCardLink.feature

```diff
@@ -7,7 +9,7 @@ Feature: Course Board - To share a board card link
         Given I am logged in as a '<teacher_2>' at '<namespace>'
         Given I am logged in as a '<teacher_1>' at '<namespace>'
 
-        # pre-condition: first teacher creates a course
+        # pre-condition: first teacher creates a course with the two teachers assigned
```

Comment:
rephrase this sentence

Reply:
changed this phrase back the way it is now.